### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by oscarpeli** [Drjacky/ImagePicker#13]
Hi 
How can I localize words on the choose dialog?
<img width="257" alt="Schermata 2021-04-12 alle 15 58 49" src="https://user-images.githubusercontent.com/47815606/114406681-04aa1b00-9ba8-11eb-8974-6230e2754a5e.png">

Thanks

**Comment by Drjacky** [comment]
Hi @oscarpeli 
What language do you need to be supported?
I need to add.

Please translate these ones in your desire language:
https://github.com/Drjacky/ImagePicker/blob/master/imagepicker/src/main/res/values/strings.xml
+
```xml
    <string name="ucrop_label_original">Original</string>
    <string name="ucrop_label_edit_photo">Edit Photo</string>

    <string name="ucrop_menu_crop">Crop</string>

    <string name="ucrop_rotate">Rotate</string>
    <string name="ucrop_scale">Scale</string>
    <string name="ucrop_crop">Crop</string>
```
And paste it here as a comment.

**Or**
Fork my project and translate by yourself and send the pull-request to me so, I can merge your change directly to the library.

**Comment by oscarpeli** [comment]
I need Italian language.
Here is the translated file.
Thanks

[strings-it.txt](https://github.com/Drjacky/ImagePicker/files/6302642/strings-it.txt)

**Comment by Leeeeerich** [comment]
Hey! And add, please Swedish.

[strings-sv.txt](https://github.com/Drjacky/ImagePicker/files/6309703/strings-sv.txt)

**Comment by Drjacky** [comment]
@Leeeeerich Sorry it's late. I already released it.
Please create a new issue regards that.

@oscarpeli It's available on [v1.8.3](https://github.com/Drjacky/ImagePicker/releases/tag/1.8.3)

**Comment by Drjacky** [comment]
@Leeeeerich Swedish translation is available on [v1.8.4](https://github.com/Drjacky/ImagePicker/releases/tag/1.8.4)
